packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -16,6 +17,16 @@ interface Props {
   setScrollTop: (event: any) => void;
   autoHeightMin?: number | string;
   updateAfterMountMs?: number;
+  scrollbarStickLeft?: boolean;
```

Comment:
What does this property do? and why was it added?

Reply:
ah, I see now.

Reply:
It replaces the former `page` I used earlier.

Reply:
Basically, It makes the page scrollbar stick to the left side to aid in hitting it.

Reply:
https://github.com/grafana/grafana/blob/10aabe8ce34656e1e3d669afeae08dd41f70c048/packages/grafana-ui/src/components/CustomScrollbar/_CustomScrollbar.scss#L51

Reply:
left? you mean right?

"isPageScrollbar"  property name would make more sense

Reply:
I think so too. 
Would update right away